src/core/index.md

```diff
@@ -1,63 +1,61 @@
 # Core
 
-DataJoint Core projects are fully open-source and are built to develop, define, manage,
-and visualize [data pipelines](./getting-started/data-pipelines). Below are the projects that
-make up the family of core open-source projects.
+DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.
 
 ## APIs
 
 <div class="grid cards" markdown>
 
--   :fontawesome-brands-python:{ .lg .middle } **DataJoint Python**
+-   :fontawesome-brands-python:{ .lg .middle } **DataJoint for Python**
 
     ---
 
-    A low-level client for managing [data pipelines](./glossary#data-pipeline).
+    A Python client for defining, operating, and querying data pipelines.
```

Comment:
Not sure I understand the purpose of mentioning Python here multiple times. It is in the title of the card. Why do we need to repeat this? Also, we should like to the glossary the usage of `data pipelines`.

Reply:
TBD in subsequent PR's.